dbrx_langchain.py

```diff
@@ -15,16 +13,15 @@
 LANCE_URI = pathlib.Path("/vectore_store")
 
 
-# NOTE: switched to snapshot_download, moved out of Cls, still downloaded in build phase
 def download_model_to_folder():
     import os
-
     from huggingface_hub import snapshot_download
     from transformers import AutoTokenizer
 
-    os.makedirs(MODEL_DIR, exist_ok=True)
     hf_token = os.environ["HF_TOKEN"]
 
+    os.makedirs(MODEL_DIR, exist_ok=True)
+
     snapshot_download(
         BASE_MODEL,
         local_dir=MODEL_DIR,
```

Comment:
@raghavdixit99 feedback on this - we should include as separate fields the new and old code so that it knows what the modifications are. just the diff formats doesn't seem to be enough